test/test.cpp

```diff
@@ -332,3 +333,105 @@ TEST(accuracy, accuracy_norm_throws_when_bad_pointer) {
 }
 
 // ==========================
+
+// ==========================
+// Throughput tests
+
+template <typename T>
+std::vector<T> matrix_sum(const std::vector<T> &first,
+                          const std::vector<T> &second) {
+  std::vector<T> res(first);
+  for (size_t i = 0; i < first.size(); i++) {
+    res[i] += second[i];
```

Comment:
please use `std::accumulate`

Reply:
I thought `std::accumulate` calculates sum in range, not sum of vectors

Reply:
sorry, my mistake - not `std::accumulate`, use `std::transform`